templates/index.spec.ts

```diff
@@ -1,12 +1,24 @@
 /**
- * @jest-environment jsdom
+ * An example test suite outlining the usage of
+ * `describe()`, `beforeEach()`, `test()` and `expect()`
+ *
+ * @see https://jestjs.io/docs/getting-started
  */
 
-import '@testing-library/jest-dom/extend-expect';
-import { render } from '@testing-library/svelte';
-import index from './index.svelte';
+describe('Index', () => {
+
+  let isTestSuitePassing = false;
+
+  beforeEach(() => {
+    isTestSuitePassing = true;
+  });
+
+  describe('isTestSuitePassing', () => {
+
+    test('should be true', () => {
+      expect(isTestSuitePassing).toBe(true);
+    });
+
+  });
 
-test('shows proper heading when rendered', () => {
-  const { getByText } = render(index);
-  expect(getByText('Hello world!')).toBeInTheDocument();
```

Comment:
I liked that this was testing the `.svelte` file before, but it was challenging that the content was not the same in the two SvelteKit templates. Perhaps we can update the SvelteKit templates so that there's some piece of shared content to solve that?

Reply:
Perhaps we could just test for `Sveltekit`?  That would pass both templates.

Reply:
That sounds like a good solution to me!

Reply:
Yeah so I can rework the tests slightly to just look for "Sveltekit".

For non Jest DOM users, we can just use the `toBeDefined()` matcher on the result of `getByText()`, and similarly continue to use `toBeInTheDocument()` for Jest DOM users -- does sound like a better solution?

Reply:
yeah, that sounds like a good idea